tests/integration/acp/p2p/replicator_test.go

```diff
@@ -89,3 +81,113 @@ func TestACP_P2POneToOneReplicatorWithPermissionedCollection_Error(t *testing.T)
 
 	testUtils.ExecuteTestCase(t, test)
 }
+
+func TestACP_P2POneToOneReplicatorWithPermissionedCollection_SourceHubACP(t *testing.T) {
+	test := testUtils.TestCase{
+		SupportedACPTypes: immutable.Some(
+			[]testUtils.ACPType{
+				testUtils.SourceHubACPType,
+			},
+		),
+		Actions: []any{
+			testUtils.RandomNetworkingConfig(),
+			testUtils.RandomNetworkingConfig(),
+			testUtils.AddPolicy{
+				Identity: immutable.Some(1),
+				Policy: `
+                    name: test
```

Comment:
suggestion: this seems like a regular policy and doesn't really improve testing readability. Would be nice to extract it to a const.

Reply:
Going to shrug this one off as personal preference, as I prefer it inline - partly so that I can see that it is just a regular policy.  I could store it in a var called `totallyNormalAndUnimportantUserPolicy`, but that would make me even more suspicious and would cause me to read it even more carefully than if it was inline (and I'd have to bounce around out of the test to do so) :)

Reply:
Happy as is